### ISSUE TITLE
add 32bit packages

### ISSUE THREAD

**Issue by taukakao** [Vanilla-OS/vso-image#37]
Some packages like steam-installer require 32bit packages. 

`dpkg --add-architecture i386`
does not help, since these packages don't exist in the repos.

**Comment by mirkobrombin** [comment]
Not sure if adding 32bit packages (which would increase the repo drastically) just for steam is a good idea. I would suggest the flatpak.

**Comment by rowbawts** [comment]
Yes using Steam Flatpak solves this with a lot less work and to my knowledge there's not many (if any) limitations to using the Flatpak.

**Comment by taukakao** [comment]
Many modding apps are not compatible with the flatpak yet. 
But it's true that  adding 32bit support for just steam would be quite excessive, especially since you can install it through apx.
I would leave this issue open anyway in case other apps requiring 32bit support are reported.

**Comment by taukakao** [comment]
wine requires 32bit, still not sure if it's worth it

**Comment by taukakao** [comment]
Fixed because the image now uses the original debian sid repos
